images/cellregmap/Dockerfile

```diff
@@ -1,10 +1,11 @@
-ARG DRIVER_TAG="latest"
+# ARG DRIVER_TAG="latest"
+ARG DRIVER_TAG="dev"
```

Comment:
This is the analysis-runner DRIVER_TAG, uses below, don't change this.

Reply:
ok, where am i adding a dev tag then?

Reply:
On deploy